uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/controllers/sub-controllers/toil-request-ctrl.spec.js

```diff
@@ -86,6 +87,7 @@
         spyOn(AbsenceTypeAPI, 'all').and.callThrough();
         spyOn(AbsenceTypeAPI, 'calculateToilExpiryDate').and.callThrough();
         spyOn(TOILRequestInstance, 'init').and.callThrough();
+        spyOn(AbsenceType, 'canExpire').and.returnValue($q.resolve(true));
```

Comment:
Changed the underlying query using `{ 'IS NOT NULL': 1 }`. This allows to use `.and.callThrough()` here and the `canExpire` code makes more sense.